src/pages/myGeotab/guides/concepts.tsx

```diff
@@ -0,0 +1,1265 @@
+        </p>
+        <CodeSample
+        language="javascript"
+        code={`var calls = [
+    call-a, // ran successfully
+    call-b, // error occurred, throw and return error
+    call-c  // never ran
+]`
+}></CodeSample>
+        <p>
+            Below is an example of the error result. The <code className="small-code-sample">requestIndex</code> property contains the index of the call that failed.
+        </p>
+        <CodeSample
+        language="javascript"
+        code={`results = {
+    "error": {
+        "message": "The method 'Foobar' could not be found. Verify the method name and ensure all method parameters are included.",
+        "code": -32601,
+        "data": {
+            "id":"2901ac83-0d7f-41a1-9cca-fd4a68e77ae7",
+            "type":"MissingMethodException",
+            "requestIndex": 1
+        }
+    },
+    "jsonrpc":"2.0"
+}`
+}></CodeSample>
+        <p>Alternatively, a successful MultiCall would look similar to:</p>
+        <CodeSample
+        language="javascript"
+        code={`calls = [
+    call-a, // ran successfully
+    call-b, // ran successfully
+    call-c  // ran successfully
+]
+results = {
+    "results": [
+        [...],
+        [...],
+        [...]
+    ]
+}`
+}></CodeSample>
+    </div>
+);
+
+//ToDo Update Reference Page Link
+const APIClientSupport: ReactNode = (
+    <div className="paragraph" id="APIClientSupport">
+        <p>
+            All of the <a href="https://geotab.github.io/sdk/software/api/clients/">API</a> clients have native support for making multi-calls. Below are examples of making multi-calls using the Javascript and .Net wrappers:    
+        </p>
+        <p>
+            JavaScript API multi-call example:    
+        </p>
+        <CodeSample
+        language="javascript"
+        code={`var calls = [
+    ["Get", { typeName: "Diagnostic" }],
+    ["Get", { typeName: "Source", search: {id: "SourceGeotabGoId"}}],
+    ["Get", { typeName: "Controller" }]
+];
+
+api.multiCall(calls, function (results) {
+    var diagnostics = results[0];
+    var sources = results[1];
+    var controllers = results[2];
+}, function (errorString) {
+    alert(errorString);
+});`
+}></CodeSample>
+        <p>
+            .Net nuget package multi-call example:
+        </p>
+        <CodeSample
+        language="javascript"
+        code={`var calls = new object[] {
+    new object[] { "Get", typeof(Diagnostic), typeof(List<Diagnostic>)},
+    new object[] { "Get", typeof(Source), new { search = new SourceSearch { Id = KnownId.SourceGeotabGoId } }, typeof(List<Source>)},
+    new object[] { "Get", typeof(Controller), typeof(List<Controller>)},
+};
+
+var results = api.MultiCall(calls);
+
+var diagnostics = (List<Diagnostic>)results[0];
+var sources = (List<Source>)results[1];
+var controllers = (List<Controller>)results[2];`
+}></CodeSample>
```

Comment:
```suggestion
}/>
```
This component can be self-closing since there are no children.